src/ui/theme/theme.tsx

```diff
@@ -14,15 +16,19 @@ const theme = createTheme({
             light: "#2C70DE",
         },
         secondary: {
-            main: "#f50057",
+            main: "#1f4076",
```

Comment:
We perhaps could have a rule for the notation #xxxxxx =>  Uppercase or LowerCase ?